KsApi/ServiceTests.swift

```diff
@@ -120,55 +120,114 @@ final class ServiceTests: XCTestCase {
     XCTAssertNil(result)
   }
 
-  func test_Array_SwiftDecodeModel_ValidModel() {
```

Comment:
Github is displaying this weirdly because I deleted an old test and added a new test.